packages/preview/cades/0.2.0/README.md

```diff
@@ -0,0 +1,33 @@
+# [Cades](https://github.com/Midbin/cades)
+
+Draw QR codes in typst.
+
+```typ
+#import "@preview/cades:0.1.0": qr-code
```

Comment:
There seems to be a mismatch between the versions here. 0.1.0 here, but 0.2.0 in the PR.